app/code/core/Mage/Directory/data/directory_setup/data-upgrade-1.6.0.3-1.6.0.4.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Directory
+ * @copyright  Copyright (c) 2023 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/** @var Mage_Core_Model_Resource_Setup $installer */
+$installer = $this;
+
+// The Netherlands Antilles (AN) was divided into Bonaire, Saint Eustatius and Saba (BQ, BES, 535),
+// Curaçao (CW, CUW, 531) and Sint Maarten (Dutch part) (SX, SXM, 534).
+//
+// See: https://www.iso.org/obp/ui/#iso:code:3166:AN
+$installer->getConnection()->delete(
+    $installer->getTable('directory/country'),
+    ['country_id = ?' => 'AN']
```

Comment:
are we safe to delete this record? should we do something for the people that have some settings dependent on it?

Reply:
I would like to avoid deleting country codes on existing instances. There might happen a lot of surprises, or shops simply continue to use them 👀

maybe its better to just remove the adding, so it only affects new instances

Reply:
I know it's probably not super-mega-clean but I tend to agree with @Flyingmana

Reply:
The only concern is only for the information collected in the past, not for the future. I have not evaluated what happens if it is removed, it is no longer displayed in the orders, customer accounts? If there are no issues it can be removed. In addition, I don't know more about that country, one thing is to delete the USA and another country AN. Has anyone used this country in OpenMage?

Reply:
I'll update it.